dashboard/config/levels/custom/music/Music Predict Level 2.level

```diff
@@ -79,9 +79,13 @@
       ],
       "codeEditableAfterSubmit": false
     },
+    "exemplar_settings": {
+      "playerEnabled": false,
+      "validationEnabled": false
+    },
```

Comment:
This field didn't exist yet when I first created this level. These are just default values.